recipe/build.sh

```diff
@@ -3,6 +3,9 @@
 echo "OS is:"
 echo "`uname`"
 
+# Tudat dependencies
+conda install numpy scipy matplotlib tqdm -y
```

Comment:
Hi @alopezrivera I don't think this should be added here, but rather in the recipe.yamp, to ensure consistency

Reply:
Great to hear from you! Where did you push the changes?

Reply:
Great to hear from you! And thanks :)